crates/accelerate/src/two_qubit_decompose.rs

```diff
@@ -182,9 +182,25 @@ fn decompose_two_qubit_product_gate(
     }
     l.mapv_inplace(|x| x / det_l.sqrt());
     let phase = det_l.arg() / 2.;
+
     Ok((l, r, phase))
 }
 
+#[pyfunction]
+#[pyo3(signature = (special_unitary))]
```

Comment:
If you do this you can drop the `py_` prefix when calling this from Python. as it sets the name of the python side to what's specified in the macro. As for the signature field we don't really need to specify this because it being a required positional named argument is the default behavior for `pyfunction`s

```suggestion
#[pyo3(name = "decompose_two_qubit_product_gate")]
```